viewer/lib/components/spec_file.dart

```diff
@@ -21,7 +21,7 @@ import 'package:registry/registry.dart';
 import 'package:split_view/split_view.dart';
 import 'package:url_launcher/url_launcher.dart';
 import 'package:flutter/foundation.dart';
-import 'dart:html' as html;
```

Comment:
This library is only available for web builds, so I moved it inside a file where it can be conditionally included.